src/main/java/org/sonar/plugins/buildbreaker/BuildBreakerPlugin.java

```diff
@@ -58,6 +58,11 @@
     name = "Forbidden configuration parameters",
     description = "Comma-separated list of <code>key=value</code> pairs that should break the build.",
     global = true,
+    project = false),
+  @Property(key = BuildBreakerPlugin.ALTERNATIVE_SERVER_URL_KEY,
+    name = "Alternative server url",
+    description = "Url to use for REST requests instead of the default sonar.core.serverBaseURL",
```

Comment:
Please change the description string to:

`"URL to use for web service requests.  If unset, uses the <code>serverUrl</code> property from <code>${sonar.working.directory}/report-task.txt</code>."`

Use string concatenation to break this long string, ensuring lines do not exceed 120 characters.